lib/govuk_content_models/version.rb

```diff
@@ -1,4 +1,4 @@
 module GovukContentModels
   # Changing this causes Jenkins to tag and release the gem into the wild
-  VERSION = "44.0.0"
+  VERSION = "44.1.1"
```

Comment:
If you are bumping the patch version, then you should probably keep the previous minor version, so `44.0.1`.